.travis.yml

```diff
@@ -19,9 +19,11 @@ before_script:
   - cd $TRAVIS_BUILD_DIR/src/parquet_thrift && thrift --gen rs parquet.thrift
   - pip install 'travis-cargo<0.2' --user && export PATH=$HOME/.local/bin:$PATH
   - cargo install cargo-travis --force && export PATH=$HOME/.cargo/bin:$PATH
+  - rustup component add rustfmt-preview
 
 script:
   - cd $TRAVIS_BUILD_DIR
+  - travis-cargo fmt -- --write-mode=diff
```

Comment:
What happens if the PR fails the style check? will the travis job fail?

Reply:
Yes, the travis job should fail, this command returns non-zero exit code when there are failures. I tested locally, will do the same on travis.